server/service/integration_enterprise_test.go

```diff
@@ -17634,6 +17634,35 @@ done
 	require.Equal(t, pkgURL, stResp.SoftwareTitle.SoftwarePackage.URL)
 	require.Equal(t, file.GetInstallScript("pkg"), stResp.SoftwareTitle.SoftwarePackage.InstallScript)
 	require.Equal(t, expectedUninstallScript, stResp.SoftwareTitle.SoftwarePackage.UninstallScript)
+
+	// Add another installer just to validate that only relevant installer bytes are removed
```

Comment:
Not following here. Looks like the cleanup removes all installers, including the Vim one?

Reply:
this was a goof! I wanted to find a way to remove just the installers added in this test, but I didn't find a good way of doing it so I just removed the assertion that checked the # of installers removed (it doesn't really matter what the number is for this test). But I forgot to remove this piece of that attempt.

Reply:
Got it. Will review again once I see a commit that either cleans this comment up or a comparable tweak.